commands/command_push.go

```diff
@@ -19,6 +19,7 @@ var (
 	pushDryRun       = false
 	pushDeleteBranch = "(delete)"
 	pushObjectIDs    = false
+	pushAllToRemote  = false
```

Comment:
I wanted to distinguish this from `fetch --all`, which downloads EVERYTHING. Though `pushAllForRef` would be a more accurate var name I think.